static-assets/components/cstudio-contextual-nav/wcm-site-dropdown-mods/wcm-root-folder.js

```diff
@@ -890,17 +900,32 @@
                     nodeSpan.setAttribute("title", treeNodeTO.title);
                     nodeSpan.className = treeNodeTO.style + " yui-resize-label treenode-label over-effect-set";
 
+                    nodeSpan.className = highlight ? nodeSpan.className + ' highlighted' : nodeSpan.className;
+
+                    if(!isLevelDescriptor){
+                        nodeSpan.dataset.uri = treeNodeTO.uri;
+                    }
+
                     treeNodeTO.html = nodeSpan;
                     var treeNode = new YAHOO.widget.HTMLNode(treeNodeTO, root, false);
 
                     treeNode.html.id = "ygtvlabelel" + treeNode.index;
                     treeNode.labelElId = "ygtvlabelel" + treeNode.index;
                     treeNode.nodeType = "CONTENT";
                     treeNode.treeNodeTO = treeNodeTO;
-                    // treeNode.initContent = treeNodeTO;
                     treeNode.renderHidden = true;
                     treeNode.nowrap = true;
 
+                    if(highlight){
+                        function loadedNode() {
+                            window.setTimeout(function(){
+                                self.scrollToHighlighted();
+                            }, 500);
+                        };
+
+                        treeNode.html.onload = loadedNode();
```

Comment:
Is this really what you wanted to do?

You're essentially setting `.onload = undefined` (setting the `.onload` to the return value of the `loadedNode` function which will always be undefined.